apiserver/Dockerfile.apiserver

```diff
@@ -1,6 +1,8 @@
 FROM hail-base
 
 COPY apiserver /apiserver
+COPY hailjwt /hailjwt
+RUN python3 setup.py /hailjwt
```

Comment:
I'm confused by this.  Who's setup.py is this?  You're not copying the one from hailjwt anymore.

Reply:
fixed